pkg/kubelet/hyper/hyper.go

```diff
@@ -812,18 +885,12 @@ func (r *runtime) RunPod(pod *api.Pod, restartCount int, pullSecrets []api.Secre
 	}
 
 	// Create and start hyper pod
-	podSpec, err := r.getPodSpec(podFullName)
```

Comment:
You generate a right `podSpec` in `buildHyperPod` in `/var/lib/kubelet/hyper/pod_name`, that's good.

Then you call `r.networkPlugin.SetUpPod`, this process is expected to change the content of `pod_name` file. (fill generated IP in that file)

So before call `r.hyperClient.CreatePod(podSpec)`, you need to load that file as new `podSpec`.

cc @YaoZengzeng